typings/objection/index.d.ts

```diff
@@ -106,7 +108,7 @@ declare namespace Objection {
 
   export interface RelationMapping {
     relation: Relation;
-    modelClass: typeof Model | string;
+    modelClass: ModelClass<any> | string;
```

Comment:
this and `Mixin` were the only two consequential updates here. Everything else is from `prettier`.